libs/langchain-google-common/src/experimental/utils/media_core.ts

```diff
@@ -0,0 +1,527 @@
+import { v1, v4 } from "uuid"; // FIXME - it is importing the wrong uuid, so v6 and v7 aren't implemented
```

Comment:
The version of uuid it is loading is from langchainjs/node_modules/@types, which still seems to be an older version, despite moving to the latest version in the main branch. So I've commented out the v6 and v7 support for now.

Reply:
Weird - we can also update globally

Reply:
The global version was updated, I thought. But it was still pulling in the wrong one for reasons I didn't understand